include/rdma/fi_eq.h

```diff
@@ -403,6 +404,8 @@ fi_cq_readfrom(struct fid_cq *cq, void *buf, size_t count, fi_addr_t *src_addr)
 static inline ssize_t
 fi_cq_readerr(struct fid_cq *cq, struct fi_cq_err_entry *buf, uint64_t flags)
 {
+	/* For compatibility with older providers. */
+	buf->src_addr = FI_ADDR_NOTAVAIL;
```

Comment:
This is tripping a CXI provider test failure where buf is NULL. I am going to update to:
```
if (buf)
    buf->src_addr = FI_ADDR_NOTAVAIL
```